script.js

```diff
@@ -12,18 +12,15 @@ fetch('https://flipsum-ipsum.net/api/icw/v1/generate?ipsum=recipe-ipsum-text-gen
         paragraphs.push(paragraphWordsArray)
```

Comment:
instead of making the array `paragraphs`, pushing 4 things to it, and later concatenating them all into one array of words, maybe you can just make the array of words in the first place? i think `paragraphs` is a bit of a remnant from the previous paragraph-based logic

Reply:
Done!